components/tools/OmeroJava/test/integration/AbstractServerImportTest.java

```diff
@@ -70,7 +71,7 @@ protected ImportLocation importFileset(List<String> srcPaths) throws Exception {
      * @throws Exception
      *             unexpected
      */
-    protected ImportLocation importFileset(List<String> srcPaths, int numberToUpload) throws Exception {
+    protected ImportLocation importFileset(List<String> srcPaths, int numberToUpload, IObject targetObject) throws Exception {
```

Comment:
Fixed in https://github.com/openmicroscopy/openmicroscopy/pull/5159/commits/03c450801095d4b7cbdef305454f58aeb5d45114